cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -1311,6 +1311,41 @@ def _validate(self, imds_secured: bool, scheduler: str):
             )
 
 
+class HeadNodeInstanceTypeValidator(Validator):
```

Comment:
This should probably be more specific that just `instance type` and mention memory in the name so folks know what its checking

Reply:
Done